static/js/src/latest-news.js

```diff
@@ -0,0 +1,18 @@
+import {fetchLatestNews} from '@canonical/latest-news';
+
+// Pull latest news from blog feed
+const articlesContainerSelector = '#latest-news-container';
+const articleTemplateSelector = '#articles-template';
+
+const articlesContainer = document.querySelector(articlesContainerSelector);
+const articlesTemplates = document.querySelector(articleTemplateSelector);
+
+if (articlesContainer && articlesTemplates) {
```

Comment:
We don't want to suppress the warning with this check. This check exists in the module and warns that it was use incorrectly. If we do this we are hiding issues across our codebases.

Reply:
Ok, removed